go.mod

```diff
@@ -176,6 +176,7 @@ require (
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/google/cel-go v0.13.0 // indirect
 	github.com/google/gnostic v0.6.9 // indirect
+	github.com/google/go-github/v50 v50.1.0 // indirect
```

Comment:
the vendor dir is missing this dependency, that's why StaticChecks are failing

Reply:
and e2e test is failing because of it xD
```
# command-line-arguments
scalers/github_runner/github_runner_test.go:***4:2: cannot find package "." in:
	/__w/keda/keda/vendor/github.com/google/go-github/v50/github
FAIL	command-line-arguments [setup failed]
```

Reply:
that'll blow it or work then :)